sopel/modules/find.py

```diff
@@ -20,7 +20,15 @@
 
 
 def setup(bot):
-    bot.memory['find_lines'] = SopelMemory()
+    if 'find_lines' not in bot.memory:
+        bot.memory['find_lines'] = SopelMemory()
+
+
+def shutdown(bot):
+    try:
+        del bot.memory['find_lines']
+    except KeyError:
+        pass
```

Comment:
This one might make sense to delete, because if the plugin misses any lines while it's inactive it could behave unexpectedly the next time someone tries to make a substitution.